src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -521,8 +521,8 @@ public IClassificationLoss CreateComponent(IHostEnvironment env)
         ///  Predict a target using a linear binary classification model trained with the <see cref="Trainers.LbfgsLogisticRegressionBinaryTrainer"/> trainer.
```

Comment:
>   [](start = 11, length = 2)

Create <see cref="LbfgsLogisticRegressionBinaryTrainer"/>, which predicts a ...

Reply:
This is binary classifier. I shouldn't touch it. Thanks!

---
In reply to: [277117848](https://github.com/dotnet/machinelearning/pull/3389#discussion_r277117848) [](ancestors = 277117848)